resources/gbxml/3997_WindowScaling_bug.xml

```diff
@@ -0,0 +1,252 @@
+  <Campus id="Facility">
+
+    <Description>A gbxml MCVE to illustrate issue 3997 where windows are incorrectly placed</Description>
+
+    <Building id="1">
+      <Area>200000</Area>
+      <BuildingStorey id="storey-1">
+        <Name>storey 1</Name>
+        <Level>0</Level>
+      </BuildingStorey>
+      <Space id="space-1" zoneIdRef="zone-1" buildingStoreyIdRef="storey-1" conditionType="HeatedAndCooled">
+        <Name>storey-1-space-1</Name>
+        <Description>internal space 1</Description>
+        <Area>11040</Area>
+        <Volume>165600</Volume>
+      </Space>
+    </Building>
+
+    <Surface surfaceType="ExteriorWall" id="surface-19">
+      <Name>storey-1-exterior-wall-1-space-1</Name>
+      <RectangularGeometry>
+        <Azimuth>303</Azimuth>
+      </RectangularGeometry>
+      <CADOjectId>none</CADOjectId>
+      <AdjacentSpaceId spaceIdRef="space-1"/>
+      <PlanarGeometry>
+        <PolyLoop>
+          <CartesianPoint>
+            <Coordinate>158</Coordinate>
+            <Coordinate>-316.4557</Coordinate>
+            <Coordinate>0</Coordinate>
+          </CartesianPoint>
+          <CartesianPoint>
+            <Coordinate>158</Coordinate>
+            <Coordinate>-316.4557</Coordinate>
+            <Coordinate>15</Coordinate>
+          </CartesianPoint>
+          <CartesianPoint>
+            <Coordinate>-158</Coordinate>
+            <Coordinate>-316.4557</Coordinate>
+            <Coordinate>15</Coordinate>
+          </CartesianPoint>
+          <CartesianPoint>
+            <Coordinate>-158</Coordinate>
+            <Coordinate>-316.4557</Coordinate>
+            <Coordinate>0</Coordinate>
+          </CartesianPoint>
+        </PolyLoop>
+      </PlanarGeometry>
+      <Opening openingType="FixedWindow" id="opening-1">
+        <RectangularGeometry>
+          <Azimuth>303</Azimuth>
+        </RectangularGeometry>
+        <PlanarGeometry>
+          <PolyLoop>
+            <CartesianPoint>
+              <Coordinate>139.2895</Coordinate>
+              <Coordinate>-316.4557</Coordinate>
+              <Coordinate>13.2</Coordinate>
+            </CartesianPoint>
+            <CartesianPoint>
+              <Coordinate>97.7105</Coordinate>
+              <Coordinate>-316.4557</Coordinate>
+              <Coordinate>13.2</Coordinate>
+            </CartesianPoint>
+            <CartesianPoint>
+              <Coordinate>97.7105</Coordinate>
+              <Coordinate>-316.4557</Coordinate>
+              <Coordinate>1.8</Coordinate>
+            </CartesianPoint>
+            <CartesianPoint>
+              <Coordinate>139.2895</Coordinate>
+              <Coordinate>-316.4557</Coordinate>
+              <Coordinate>1.8</Coordinate>
+            </CartesianPoint>
+          </PolyLoop>
+        </PlanarGeometry>
+      </Opening>
+    </Surface>
```

Comment:
The surface and subsurface we care about. Rest of the gbxml file is just the floor/ceilings and other walls, so the space is bounded.